js/src/forum/index.js

```diff
@@ -50,5 +51,60 @@ app.initializers.add('flarum/nicknames', () => {
     }
   });
 
```

Comment:
Wouldn't it be easier to run `if (app.forum.attribute('displayNameDriver') !== 'nickname') return;` here rather than in all the `extend` blocks?

Reply:
Initializers run before `app.forum.attribute` is initiated, so unfortunately that isnt possible.

Reply:
Oh of course, I always forget that.